sdk/appcenter-distribute/src/main/java/com/microsoft/appcenter/distribute/Distribute.java

```diff
@@ -539,6 +539,12 @@ private boolean tryResetWorkflow() {
         return false;
     }
 
+    @Override
+    public void onActivityStarted(Activity activity) {
+        super.onActivityStarted(activity);
+        registerReceiver(activity);
```

Comment:
Now that we register receiver in "onStart" callback, Do we need unregister it in "onStop" so that it comes in pairs?

Reply:
No, it doesn’t depend on lifecycle of application. Receiver is registered only when distribute is enabled and is unregistered when distribute is disabled. This call need in this method to avoid issue when distribute was enabled, but activity has not started yet. If receiver was already registered so this call will be ignored.